StatPlugin.cs

```diff
@@ -1,40 +1,36 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using ServiceStack.Redis;
 
 namespace DNWS
 {
   class StatPlugin : IPlugin
   {
-    protected static Dictionary<String, int> statDictionary = null;
+    protected static RedisManagerPool redisManager = null;
     public StatPlugin()
     {
-      if (statDictionary == null)
-      {
-        statDictionary = new Dictionary<String, int>();
-
-      }
+        redisManager =  new RedisManagerPool("redis:6379");
```

Comment:
it's weird that your redisManager is static, i.e., it's belong to the class, but you instantiate new RedisManagerPool every time you instantiate new object. You better use Singleton pattern for this.